documentation/autoinstall-reference.md

```diff
@@ -411,6 +411,23 @@ A list of SSH public keys to install in the initial user's account.
 **type:** boolean
 **default:** `true` if `authorized_keys` is empty, `false` otherwise
 
+<a name="codecs"></a>
```

Comment:
Would you merge the documentation directory to ubuntu/kinetic?  As we add main branch features, we should not use that as the basis for the documentation on Discourse.

Reply:
Good point. I've just opened https://github.com/canonical/subiquity/pull/1474.